ci/bazel-scripts/main.sh

```diff
@@ -10,8 +10,18 @@ set -eufo pipefail
 ic_version_rc_only="0000000000000000000000000000000000000000"
 s3_upload="False"
 
+protected_branches=("master" "rc--*" "hotfix--*" "master-private")
```

Comment:
Hotfix branches start with `hotfix-` (i.e. single `-`):
```suggestion
protected_branches=("master" "rc--*" "hotfix-*" "master-private")
```